src/lib/project.js

```diff
@@ -88,7 +88,11 @@ async function project({ name, ui }) {
 
     // Add SnarkyJS as a dependency in the UI project.
     let pkgJson = fs.readJSONSync(path.join('ui', 'package.json'));
-    pkgJson.dependencies.snarkyjs = '0.*';
+    // Add dependencies object if none is found in the package.json because generated
+    // Svelte projects do not have dependencies included.
+    pkgJson.dependencies
```

Comment:
Personal preference, but I find this more clear. Not a strong opinion

```js
if (!pkgJson.dependencies) pkgJson['dependencies'] = {}
pkgJson.dependencies.snarkyjs = '0.*';
```

Reply:
Agreed this is cleaner.